checkstyle.xml

```diff
@@ -52,16 +52,6 @@ page at http://checkstyle.sourceforge.net/config.html -->
       <property name="severity" value="error"/>
     </module>
 
-    <module name="ImportOrder">
-      <!-- Checks for out of order import statements. -->
-
-      <property name="severity" value="warning"/>
-      <property name="groups" value="com.google,android,junit,net,org,java,javax"/>
```

Comment:
I think you need to delete  line 59 and keep the others.

Reply:
Done.